src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
       systems:
         type: object
         description: Map[String, SubsystemStatus]
+        default: {
```

Comment:
See above. Also I'm wondering if this is correct YAML syntax? For a string spanning several lines could use
```
default: >
  This
  is
  a 
  long 
  string
```